code/__DEFINES/status_effects.dm

```diff
@@ -39,6 +39,8 @@
 
 #define STATUS_EFFECT_DASH /datum/status_effect/dash // Grants the ability to dash, expiring after a few secodns
 
+#define STATUS_EFFECT_FLESHMEND /datum/status_effect/fleshmend //Rapid burn/brute/oxy/blood healing from the cling ability
```

Comment:
```suggestion
/// Rapid burn/brute/oxy/blood healing from the cling ability
#define STATUS_EFFECT_FLESHMEND /datum/status_effect/fleshmend 
```
this should follow comment style for autodoc comments

Reply:
erm... so not the same sort of comment as everything else around it in that file?

Reply:
yeah, you don't have to touch the other stuff, but moving stuff to autodoc format is preferable